tianshou/utils/net/common.py

```diff
@@ -289,20 +326,30 @@ def __init__(
     def forward(
         self,
         obs: Union[np.ndarray, torch.Tensor],
-        state: Optional[Dict[str, torch.Tensor]] = None,
-        info: Dict[str, Any] = {},
+        state: Optional[Union[RecurrentStateBatch, Dict[str, torch.Tensor]]] = None,
+        **kwargs: Any,
     ) -> Tuple[torch.Tensor, Dict[str, torch.Tensor]]:
         """Mapping: obs -> flatten -> logits.
 
         In the evaluation mode, `obs` should be with shape ``[bsz, dim]``; in the
         training mode, `obs` should be with shape ``[bsz, len, dim]``. See the code
         and comment for more detail.
+
+        :param obs:
```

Comment:
missing docstring?

Reply:
obs is described above and by the type hint, it would be just repetition imo. What would you propose to write?